components/scroll-wrapper/README.md

```diff
@@ -1,22 +1,35 @@
-# Scroll Wrapper
+# Scroll Wrapper [d2l-scroll-wrapper]
```

Comment:
I believe this will cause some weirdness with the heading levels since there will be no `h2`, just `h1` then `h3`s

Reply:
Any ideas on a top level heading for this page? I didn't want to just duplicate the `Scroll Wrapper` twice in an H1/H2

Reply:
Could be something like `Scrolling Container`

Reply:
We could also modify documentation to have a heading level option

Reply:
Ended up adding a new H1 and changing the H2 text to say `Horizontal scroll wrapper`, I think that should work alright